apps/prairielearn/python/prairielearn.py

```diff
@@ -1093,7 +1093,7 @@ def string_to_number(s, allow_complex=True):
         return None
 
 
-def string_fraction_to_number(a_sub, allow_fractions=True, allow_complex=True):
+def string_fraction_to_number(a_sub, *, allow_fractions=True, allow_complex=True):
```

Comment:
Need to check `string_fraction_to_number` callsites outside of our codebase.

Reply:
There are two usages of this in the wild that don't use named arguments:

```
./pl-ucb-me40/elements/course-element-1/course-element-1.py:    value, newdata = pl.string_fraction_to_number(a_sub, allow_fractions, allow_complex)
./pl-calpoly-phys132/elements/pl-ccnumber-input/pl-ccnumber-input.py:    value, newdata = pl.string_fraction_to_number(a_sub, allow_fractions, allow_complex)
```

Reply:
Resolved in [a5d43a0](https://github.com/PrairieLearn/PrairieLearn/pull/11107/commits/a5d43a0e6afdee5e05486e4a0a1d65c38efc8a1a).